orchestrator-salt/src/main/resources/salt-common/salt/fluent/cron/cdp_logging_agent_check_distribute.sh

```diff
@@ -0,0 +1,2 @@
+# run logging agent script distribute every Sunday at 02:00:00
+0 2 * * SUN root sh /opt/salt/scripts/cdp_logging_agent_check.sh distribute
```

Comment:
i have discussed this part with @sodre90 
as distribute calling salt commands, it's not ideal to do that overall, (especially in a scheduled way), as if it is not triggered from CB side, it could cause problems later. It's not the case if it is triggered by diagnostics itself (as no other flow will run at that time), 
mainly this scheduled call is needed, if let's say, there is an older cluster, and it uses old salt states, we need to check the new nodes as well, so sometimes it is required to re-distribute this script to execute the checks....that could be worked around if the salt states are **upgraded** (e.g. during diagnostics), but  it's risky to do that on all of the salt states, as for an older (half-1year) cluster, we cannot be sure all the states will work as expected -> to solve this we need to be able to update only 1 or specific modules in the salt states (note this are persisted in the cluster component table) -> so if that feature is implemented, this can be removed (actually, as the cdp_logging_agent_check.sh can update itself on the minions, if the implementation for the partial salt state upgrade is done, i can change it's code to cleanup the crontab, so it won't call distribute command anymore without CB)